CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## Supported Version 1.4.0
+
+This release includes:
+- The set of instances that are associated with an ELB can now be modified.
+- Added 'ssl_certificate_id' property to elb_loadbalancer.
+
+As well as following fixes and improvements:
+- Fixed multiple issues related to the region property being displayed/returned incorrectly.
+- Documentation improvements.
+- Multiple test improvements.
+- Rubocop updates.
+- Fixed issue with elb_loadbalancer availability_zones synchronisation detection.
+- Use the VPC's default subnet when none is specified on the ec2_instance.
+- Use the region as part of the cache key in the base provider.
```

Comment:
The important bit here is "Enable `puppet resource` usage across regions for ec2_vpc (and maybe others)."